src/App.tsx

```diff
@@ -1,15 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
 export const App: React.FC = () => {
-  const count = 0;
+  const [count, addition] = useState(0);
```

Comment:
```suggestion
  const [count, setCount] = useState(0);
```
setState function should be named according to the noun variable
https://react.dev/reference/react/useState#reference